CliClient/app/command-e2ee.js

```diff
@@ -50,7 +50,15 @@ class Command extends BaseCommand {
 				this.stdout(_('Operation cancelled'));
 				return;
 			}
-
+			const password2 = await this.prompt(_('Confirm master password:'), { type: 'string', secure: true });
+			if (!password2) {
+				this.stdout(_('Operation cancelled'));
```

Comment:
Nice that you used British English which is indeed consistent with the rest of the app.